addon/components/frost-bunsen-cell/component.js

```diff
@@ -1,12 +1,11 @@
 import _ from 'lodash'
-import Ember from 'ember'
 import computed, {readOnly} from 'ember-computed-decorators'
 
-import PropTypesMixin, {PropTypes} from 'ember-frost-bunsen/mixins/prop-types'
+import FrostComponent, {PropTypes} from 'ember-frost-component'
```

Comment:
Using a mixin is certainly an option but I thought `FrostComponent.extend({` looked better than `Ember.Component.extend(PropTypesMixin, {`. Guess it comes down to the more desired DX by the masses.